plugins/systemlogs/frontend/public/javascripts/countly.views.js

```diff
@@ -1,539 +1,414 @@
-        var i;
-        if (this._query) {
-            if (this._query.a) {
-                activeAction = jQuery.i18n.prop("systemlogs.action." + this._query.a);
+(function() {
+    var FEATURE_NAME = "systemlogs";
+    var SystemLogsViewExpanded = countlyVue.views.create({
+        template: CV.T("/systemlogs/templates/logs-expanded.html"),
+        props: {
+            row: {
+                type: Object,
+                default: {}
             }
-
-            if (this._query.user_id) {
-                for (i = 0; i < meta.users.length; i++) {
-                    if (meta.users[i]._id === this._query.user_id) {
-                        activeUser = meta.users[i].full_name;
-                        break;
-                    }
+        },
+        computed: {
+            change: function() {
+                if (typeof this.row.i.before !== "undefined" && typeof this.row.i.after !== "undefined") {
+                    return true;
                 }
-            }
-        }
-
-        this.templateData = {
-            "page-title": jQuery.i18n.map["systemlogs.title"],
-            "active-action": activeAction,
-            "actions": meta.action,
-            "active-user": activeUser,
-            "users": meta.users,
-            query: this._query,
-            back: this._back
-        };
-
-        var self = this;
-        if (!isRefresh) {
-            $(this.el).html(this.template(this.templateData));
-            if (!app.hasRoutingHistory()) {
-                $(".back-link").css('display', 'none');
-            }
-
-            $(".back-link").click(function() {
-                app.back();
-            });
-
-            this.dtable = $('#systemlogs-table').dataTable($.extend({}, $.fn.dataTable.defaults, {
-                "aaSorting": [[ 1, "desc" ]],
-                "bServerSide": true,
-                "sAjaxSource": countlyCommon.API_PARTS.data.r + "?api_key=" + countlyGlobal.member.api_key + "&app_id=" + countlyCommon.ACTIVE_APP_ID + "&method=systemlogs",
-                "fnServerData": function(sSource, aoData, fnCallback) {
-                    self.request = $.ajax({
-                        "dataType": 'json',
-                        "type": "POST",
-                        "url": sSource,
-                        "data": aoData,
-                        "success": function(data) {
-                            self.loaded = true;
-                            self.request = null;
-                            fnCallback(data);
-                            CountlyHelpers.reopenRows(self.dtable, self.expandTable, self);
-                        }
-                    });
-                },
-                "fnServerParams": function(aoData) {
-                    if (self._query) {
-                        aoData.push({ "name": "query", "value": JSON.stringify(self._query) });
+                return false;
+            },
+            renderField: function(key, field, sub) {
```

Comment:
I think it would be better to move renderField fn to methods, rather than computed, as you "call" it with some args in various places.

Reply:
Also we could move this logic to template itself, since it is mostly about creating visual output.